setup.py

```diff
@@ -347,7 +347,7 @@ def get_exts_for(name):
         "cattrs<1.1; python_version<='3.6'",
         "cattrs; python_version>='3.7'",
         "six>=1.12.0",
-        "typing_extensions",
+        "typing_extensions<4.6.0",
```

Comment:
is there an actual incompatibility between ddtrace and `typing_extensions`? I thought the new version of typing_extensions was only affecting a few of our integration packages>

Reply:
Based on the commit message, I'd say not necessarily.

Reply:
Yes, this appears to be an issue with just two integrations